src/components/PurchaseOrder/Summary/TotalEncumberedValue/TotalEncumberedValue.js

```diff
@@ -0,0 +1,63 @@
+import React, {
+  useEffect,
+  useState,
+} from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+import {
+  KeyValue,
+  Loading,
+} from '@folio/stripes/components';
+import {
+  AmountWithCurrencyField,
+  transactionsManifest,
+} from '@folio/stripes-acq-components';
+
+const TotalEncumberedValue = ({ orderId, label, mutator }) => {
+  const [totalEncumbered, setTotalEncumbered] = useState();
+
+  useEffect(
+    () => {
+      setTotalEncumbered();
+
+      if (orderId) {
+        mutator.orderTransactions.GET({
+          params: {
+            query: `encumbrance.sourcePurchaseOrderId==${orderId}`,
```

Comment:
IMHO, it could be written in manifest

Reply:
Do we really need to use `batchFetch` in this case? Here is only one param - `orderId`

Reply:
right, batcFetch name is confused me, I was thinking it will request items in batch if there are a lot of them (>1000)